### PR TITLE
Add upgrade-minor job for okd-scos stable and next

### PR DESCRIPTION
**Pull request by Prashanth684**

### CODE REVIEW

File: core-services/release-controller/_releases/release-okd-scos-4.y-next.json [hunk 1]
```diff
@@ -11,9 +11,13 @@
   "verify":{
     "upgrade":{
       "upgrade":true,
-      "prowJob": {
-        "name": "release-openshift-okd-scos-installer-e2e-aws-upgrade-from-scos-next"
-      },
```
Comment (bradmwilliams) [inline core-services/release-controller/_releases/release-okd-scos-4.y-next.json:16]: Is loosing this formatting intentional?  Ultimately, it doesn't matter all that much, but it's inconsistent with the formatting in the vast majority of the other files.

File: core-services/release-controller/_releases/release-okd-scos-4.y-stable.json [hunk 2]
```diff
@@ -11,9 +11,13 @@
...
-      "prowJob": {
-        "name": "release-openshift-okd-scos-installer-e2e-aws-upgrade-from-scos-stable"
-      },
+      "prowJob":{"name":"release-openshift-okd-scos-installer-e2e-aws-upgrade-from-scos-stable"},
+      "maxRetries": 2
+    },
+    "upgrade-minor":{
+      "upgrade":true,
+      "upgradeFrom": "PreviousMinor",
+      "prowJob":{"name":"release-openshift-okd-scos-installer-e2e-aws-upgrade-from-scos-stable"},
```
Comment (bradmwilliams) [inline core-services/release-controller/_releases/release-okd-scos-4.y-stable.json:20]: Same

**Review by bradmwilliams — CHANGES_REQUESTED** [review]

File: core-services/release-controller/_releases/release-okd-scos-4.y-next.json (hunk 1 above)
Comment (Prashanth684) [reply core-services/release-controller/_releases/release-okd-scos-4.y-next.json:16]: i compared it to the stable config here: https://github.com/openshift/release/blob/master/core-services/release-controller/_releases/release-ocp-4.y-stable.json and thought I would do the same. if it is inconsistent, I can revert it to the original format.

Comment (Prashanth684) [reply core-services/release-controller/_releases/release-okd-scos-4.y-next.json]: fixed it to the original format.

**Review by bradmwilliams — APPROVED** [review]

**Pull request merged**